tests/test_main.cpp

```diff
@@ -110,6 +110,7 @@
 #include "tests/scene/test_curve_2d.h"
 #include "tests/scene/test_curve_3d.h"
 #include "tests/scene/test_gradient.h"
+#include "tests/scene/test_height_map_shape_3d.h"
```

Comment:
This needs to be moved down under `#ifndef _3D_DISABLED`

Reply:
Sorted this :smiley: